<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1929</width>
    <height>944</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans Mono</family>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>-10</y>
      <width>520</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:60px;
color:black;</string>
    </property>
    <property name="text">
     <string>体育心理测试系统</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>89</y>
      <width>1920</width>
      <height>891</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="begin">
     <widget class="QPushButton" name="toLoginBtn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>270</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toRegisterBtn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>430</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginPage">
     <widget class="QGraphicsView" name="graphicsView_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 227, 243);
border-radius:50px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>240</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>360</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>360</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmLoginBtn">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>560</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnBeginBtn">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>560</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="registerPage">
     <widget class="QGraphicsView" name="graphicsView_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 227, 243);
border-radius:50px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>320</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>400</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>单位</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>480</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>邮箱</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>240</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>320</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>400</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>470</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmRegBtn">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>590</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnBegBtn1">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>590</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adminMenu">
     <widget class="QGraphicsView" name="graphicsView_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>1000</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 227, 243);
border-radius:50px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toInfoBtn">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>580</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string> 关于软件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toDateBtn">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>320</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>数据导出</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goTestBtn">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>190</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>进入测试</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="introDuctionText">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>911</width>
        <height>61</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans Mono'; font-size:30pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;平台介绍：这是一个。。。。。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toHelpBtn">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>450</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
border-color: rgb(0, 85, 255);
</string>
      </property>
      <property name="text">
       <string>使用帮助（manual）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="helpPage">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>180</y>
        <width>441</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>使用帮助信息界面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnMenu_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="introductionPage">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>200</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;Adobe Devanagari&quot;;</string>
      </property>
      <property name="text">
       <string>软件介绍界面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnMenu_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gratitudePage">
     <widget class="QPushButton" name="returnMenu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textArea">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>651</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Adobe Devanagari','DejaVu Sans Mono'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;本项目由....完成，感谢您的使用。此外，由衷希望得到您在使用中的测试数据，实现数据共享。。。。&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/logo1.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1921</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1431</x>
      <y>19</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>当前用户:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>15</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>未登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitUserBtn">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toThankBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>致谢</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1929</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
